.github/scripts/release-plugins.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+git config --global user.email "githubbot@gluonhq.com"
+git config --global user.name "Gluon Bot"
+
+CLIENT_MAVEN_REPO_SLUG=gluonhq/client-maven-plugin
+CLIENT_GRADLE_REPO_SLUG=gluonhq/client-gradle-plugin
+
+cd $HOME
+git clone --depth 5 https://github.com/$CLIENT_MAVEN_REPO_SLUG
+git clone --depth 5 https://github.com/$CLIENT_GRADLE_REPO_SLUG
+
+###############################
+# 
+# Release client-maven-plugin
+#
+###############################
+cd $HOME/client-maven-plugin
+# Update Substrate
+mvn versions:set-property -Dproperty=substrate.version -DnewVersion=$1 -DgenerateBackupPoms=false
+git commit pom.xml -m "Update Substrate version to $1"
+# Remove SNAPSHOT
+mvn versions:set -DremoveSnapshot
+CLIENT_PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+# Commit and push tag
+git commit pom.xml -m "Release $CLIENT_PROJECT_VERSION"
+git tag $CLIENT_PROJECT_VERSION
+git push https://gluon-bot:$2@github.com/$CLIENT_MAVEN_REPO_SLUG $CLIENT_PROJECT_VERSION
+
+###############################
+# 
+# Release client-gradle-plugin
+#
+###############################
+cd $HOME/client-gradle-plugin
+# Update Substrate version
+sed -i "0,/com.gluonhq:substrate:.*/s//com.gluonhq:substrate:$1'/" build.gradle
+git commit build.gradle -m "Update Substrate version to $1"
+# Remove SNAPSHOT
+sed -i "0,/^version '.*'/s//version '$CLIENT_PROJECT_VERSION'/" build.gradle
```

Comment:
I would recalculate the value for CLIENT_PROJECT_VERSION. It is not guaranteed that the version for the maven and gradle client plugin will always be the same.

Reply:
Since, we release plugins based on Substrate releases, I would like to keep both plugin versions in sync.

Reply:
There is no guarantee that we both versions will be kept in sync as there are no rules that enforce this. It is still possible to release one plugin separately from the other. At the moment, it looks like we only release the plugins when a new version of substrate is released. But in the future, when substrate development slows down somewhat, it might be that versions between the two plugins diverge because some fixes/improvements are being brought in that don't have directly anything to do with substrate.

Reply:
When we take the route suggested by you in the future, I was hoping that the plugin releases won't be handled by Substrate and this script will be deprecated :)